node/resolvers/Queries/Users.ts

```diff
@@ -300,14 +331,19 @@ export const listUsers = async (
         'email',
         'canImpersonate',
       ],
-      schema: config.version,
       pagination: { page: 1, pageSize: 50 },
+      schema: config.version,
       ...(where && { where }),
     })
 
     return res
-  } catch (e) {
-    return { status: 'error', message: e }
+  } catch (error) {
```

Comment:
The **listUsers** query was deprecated. no need for an error log, and it can simply return a value when called